zeppelin-interpreter/src/main/java/org/apache/zeppelin/user/AuthenticationInfo.java

```diff
@@ -57,8 +58,11 @@ public AuthenticationInfo(String user, String roles, String ticket) {
     this.user = user;
     this.ticket = ticket;
     if (StringUtils.isNotBlank(roles) && roles.length() > 2) {
-      String[] r = roles.substring(1, roles.length() - 1).split(",");
-      this.roles = Arrays.asList(r);
+      this.roles = new ArrayList<>();
+      for (final String role : roles.substring(1, roles.length() - 1)
```

Comment:
not sure why roles has single-quote around it which seems weird.

Reply:
It's not single quotes but brackets, e.g. ["role1, role2"].
As far as I see, this comes from the client through the web socket connection from Method org.apache.zeppelin.socket.NotebookServer.onMessage(NotebookSocket, String). But is not related to this Issue here IMHO.